invenio_rdm_records/services/pids/manager.py

```diff
@@ -21,7 +21,7 @@
 class PIDManager:
     """RDM PIDs Manager."""
 
-    def __init__(self, providers, required_schemes=None):
+    def __init__(self, providers, required_schemes=[]):
```

Comment:
is there any reason why we changed this to mutable argument? it is considered not to be a best practice